modules/acpica/Kconfig

```diff
@@ -8,7 +8,7 @@ menu "ACPI driver support"
 
 config ACPI_DSDT_SUPPORT
 	bool  "Build source code for DSDT ACPICA support"
-	default y if PCIE
+	default y if PCIE && ACPI
```

Comment:
Shouldn't this have a `depends on ACPI`? If you have that then the original `if PCIE` should be enough

Reply:
Should it be something like
```
config ACPI
	bool

if ACPI

menu "ACPI driver support"

.....

endif 
```

Reply:
@finikorg sure, that might be better, especially if we introduce more ACPI_* options in the future

Reply:
updated

Reply:
Maybe because `#` is a comment in Kconfig?

Reply:
> Maybe because `#` is a comment in Kconfig?

yes, you are right, just missed it!

Reply:
@najumon1980 you put `menu` out of the `if`, why? Have you seen my sample above? Does it work?